CMake/External_GTest.cmake

```diff
@@ -0,0 +1,24 @@
+ExternalProject_Add(GTest
+  URL ${GTest_url}
+  URL_MD5 ${GTest_md5}
+  PREFIX ${fletch_BUILD_PREFIX}
+  DOWNLOAD_DIR ${fletch_DOWNLOAD_DIR}
+  INSTALL_DIR ${fletch_BUILD_INSTALL_PREFIX}
+  PATCH_COMMAND ${CMAKE_COMMAND}
+    -DGTest_patch:PATH=${fletch_SOURCE_DIR}/Patches/GTest
+    -DGTest_source:PATH=${fletch_BUILD_PREFIX}/src/GTest
+    -P ${fletch_SOURCE_DIR}/Patches/GTest/Patch.cmake
+  CMAKE_GENERATOR ${gen}
+  CMAKE_ARGS
+    ${COMMON_CMAKE_ARGS}
+  )
+
+set(GTEST_ROOT ${fletch_BUILD_INSTALL_PREFIX} CACHE STRING "")
+
+file(APPEND ${fletch_CONFIG_INPUT} "
+########################################
+# GTest
+########################################
+set(GTEST_ROOT \$\{fletch_ROOT\})
```

Comment:
I steered you a bit wrong the other day. I found that escaping the { } is not required and causes issues for CMake 2.8.12. Can you remove all escapes except the $ ... thanks!